terms_dynamic.go

```diff
@@ -27,9 +27,14 @@ import (
 	// will be automatically included anyway.
 	"github.com/gdamore/tcell/v2/terminfo"
 	"github.com/gdamore/tcell/v2/terminfo/dynamic"
+
+	"fmt"
```

Comment:
Normally I prefer fmt to be (standard lib) imports first, but in this case I think it might be fine because of the comment above the others.